.github/workflows/ci.yml

```diff
@@ -47,7 +48,7 @@ jobs:
             ${{ runner.os }}-maven-
       - name: Build with Maven
         run: |
-          mvn -B clean install javadoc:javadoc -Pgenerate-docs
+          mvn -B clean install javadoc:javadoc -Pgenerate-docs -Dfabric8.httpclinent.impl=${{ matrix.httpclient }}
```

Comment:
I prefer the e2e approach, unit tests mock most http interactions anyways

Reply:
Reverted in [57b341b](https://github.com/apache/flink-kubernetes-operator/pull/881/commits/57b341bd841404c4d31d083adcc0d1385f22dffc)